### ISSUE TITLE
Possible bug in NtCreateFIle

### ISSUE THREAD

**Issue by BiatuAutMiahn** [xenia-project/xenia#1382]
I discovered this when looking into why Halo 3 does not restore the last profile when restarting the game. It appears that a memory location is included in the path string when the game saves the profile. Every time the game is restarted the location is different and thus the folder name is different as well.

Here is a screenshot:
![image](https://user-images.githubusercontent.com/6149596/59563587-f7529280-9009-11e9-99b6-ba6a14857f78.png)

**Comment by BiatuAutMiahn** [comment]
Correction, the hex string looks more like a timestamp, creation time maybe?

**Comment by reaperfallen** [comment]
Yeah I think same, the game is crashing perhaps generating a random user profile... when my game gone to "Checkpoint" the game get crash.

**Comment by reaperfallen** [comment]
> Correction, the hex string looks more like a timestamp, creation time maybe?

Maybe, but it's strange, i've watching videos of people editing profiles of Halo 3 (Xbox 360) and they do not have this type of files, maybe is a issue

**Comment by BiatuAutMiahn** [comment]
Based on the screenshots in this thread [https://www.xboxchaos.com/topic/358-how-to-swap-maps-on-the-main-menu/](https://www.xboxchaos.com/topic/358-how-to-swap-maps-on-the-main-menu/) it looks like the naming scheme is accurate except, the path is declared as /content/<profile id>/<TitleID>/00000001/...

**Comment by CloudFR** [comment]
Hi all, you thinks for this bug speedly fixed ?
I have same problem, i dont load last savegame, when i restart xenia, halo 3 not loading and restart a new game.
Please fix hurry for the little problem i guess ?

**Comment by BiatuAutMiahn** [comment]
Unfortunately, if this was a simple fix it would have been done already. I speculate the bug has to do with the way Xenia handles profiles. This issue is similar to the `kernel-save-file-errors` tag we see on so many titles.

**Comment by CloudFR** [comment]
Oh is so damaged... i play one hour in halo 3 but it's crash

**Comment by Margen67** [comment]
@CloudFR We know already. Stop spamming.
